projects/libreoffice/build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -eu
+
+git pull -r
```

Comment:
This is redundant -- we do everything from scratch every time we build new binaries.